### DISCUSSION TITLE
v1.0.0

### DISCUSSION THREAD

**Discussion by AlonsoGuevara** [microsoft/graphrag#1503] [Announcements]
## What's Changed
- patch: Add Parent id to communities data model
- patch: Add migration notebook.
- patch: Create separate community workflow, collapse subflows.
- patch: Dependency Updates
- patch: cleanup and refactor factory classes.

**Full Changelog**: https://github.com/microsoft/graphrag/compare/v0.9.0...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/graphrag/releases/tag/v1.0.0'>v1.0.0</a>.</em>

**Comment by Zochory** [comment]
Hello ! Do we have any news regarding LazyGraphRag ?

**Comment by DStarEpoch** [comment]
Has anyone success run graphrag 1.0.0 with ollama local models?

**Reply by ganeshrohan** [reply]
Not working! The older versions looks fine - I had to downgrade to version 0.3.2

**Comment by retgreen98** [comment]
Exactly! The source code has changed and the former progress isnt working now
